pkg/skaffold/util/clock.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
I think this can just live in date_time.go and the tagger package, not sure what else we could use it for?

Reply:
yeah, that's where I had it the first time - but then I figured if we'd need time/date related mocking anywhere else, this could serve as a pattern, save some "time". Happy to move it back and do the extraction to util when the second use case actually occurs!